examples/blue-green/main.tf

```diff
@@ -0,0 +1,405 @@
+provider "aws" {
+  region = local.region
+}
+
+data "aws_availability_zones" "available" {}
+
+locals {
+  region = "eu-west-1"
+  name   = "ex-${basename(path.cwd)}"
+
+  vpc_cidr = "10.0.0.0/16"
+  azs      = slice(data.aws_availability_zones.available.names, 0, 3)
+
+  container_name = "ecsdemo-frontend"
+  container_port = 3000
+
+  tags = {
+    Name       = local.name
+    Example    = local.name
+    Repository = "https://github.com/terraform-aws-modules/terraform-aws-ecs"
+  }
+}
+
+################################################################################
+# Cluster
+################################################################################
+
+module "ecs" {
+  source = "../../"
+
+  cluster_name = local.name
+
+  # Cluster capacity providers
+  default_capacity_provider_strategy = {
+    FARGATE = {
+      weight = 50
+      base   = 20
+    }
+    FARGATE_SPOT = {
+      weight = 50
+    }
+  }
+
+  services = {
+    ecsdemo-frontend = {
+      cpu    = 1024
+      memory = 4096
+
+      # for blue/green deployments
+      deployment_configuration = {
```

Comment:
instead of a whole, new example - lets just add this to the existing Fargate example